packages/feedback/src/widget/Form.ts

```diff
@@ -161,28 +167,48 @@ export function Form({
       errorEl,
 
       !isAnonymous &&
-        showName &&
+        (showName || isNameRequired) &&
         createElement(
           'label',
           {
             htmlFor: 'name',
             className: 'form__label',
           },
-          [nameLabel, nameEl],
+          isNameRequired
+            ? [
+                createElement(
+                  'span',
+                  { className: 'form__label__text' },
+                  nameLabel,
+                  createElement('span', { className: 'form__label__text--required' }, ' (required)'),
+                ),
+                nameEl,
+              ]
+            : [nameLabel, nameEl],
         ),
-      !isAnonymous && !showName && nameEl,
+      !isAnonymous && !(showName || isNameRequired) && nameEl,
 
       !isAnonymous &&
-        showEmail &&
+        (showEmail || isEmailRequired) &&
         createElement(
           'label',
           {
             htmlFor: 'email',
             className: 'form__label',
           },
-          [emailLabel, emailEl],
+          isEmailRequired
+            ? [
+                createElement(
+                  'span',
+                  { className: 'form__label__text' },
+                  emailLabel,
+                  createElement('span', { className: 'form__label__text--required' }, ' (required)'),
```

Comment:
can we move the `isEmailRequired` condition to show this element only instead the the tuples? (And same thing with name)